e2e/content-services/permissions/permissions-component.e2e.ts

```diff
@@ -300,8 +298,12 @@ describe('Permissions Component', function () {
 
         afterEach(async (done) => {
 
-            await uploadActions.deleteFilesOrFolder(alfrescoJsApi, file.entry.id);
+            try {
+                await uploadActions.deleteFilesOrFolder(alfrescoJsApi, file.entry.id);
 
+            } catch (error) {
+
```

Comment:
should we console log the error?

Reply:
we usually don't do it in the e2e but I guess we could create a service to do it when the log parameter is passed in the configuration